fastn-js/README.md

```diff
@@ -0,0 +1,12 @@
+# How to Format JavaScript Files
+
+JavaScript formatting is tested using [`dprint-check-action`](https://github.com/marketplace/actions/dprint-check-action). 
+It utilizes the [dprint-prettier-plugin](https://dprint.dev/plugins/prettier/). 
+
+To format the `fastn-js` files, you can install [dprint](https://dprint.dev/install/) on your system and execute the following command from the root of the project:
+
```

Comment:
Another long line. You do not look at your PR @harshdoesdev before asking for review. First must come self review, which means open the PR and look at it. Do not let lines wrap.